server/middleware/isUserExist.js

```diff
@@ -0,0 +1,30 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import ResponseHandler from '../helpers/responseHandler';
+import {
+  UNAUTHORIZED,
+  BAD_REQUEST,
+} from '../helpers/statusCode';
+import Database from '../models/database';
+
+dotenv.config();
+const isEmployee = async (req, res, next) => {
+  const token = req.header('x-auth-token');
```

Comment:
@murera I would suggest you to rename `x-auth-token` to `authorization`, for the sake of future contributors.